packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -393,6 +398,12 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
   private _onWindowResize = (): void => {
     const scrollbarWidth = this._getScrollbarWidth();
     const scrollbarHeight = this._getScrollbarHeight();
+    const { contentContainer } = this;
+
+    if (contentContainer) {
+      this._scrollLeft = contentContainer.scrollLeft;
+      this._scrollTop = contentContainer.scrollTop;
+    }
```

Comment:
This read won't cause reflow. The first read which is _getScrollbarWidth() is likely to cause reflow... working on finding a way to avoid/min reflow.